services/QuillLMS/app/controllers/api/v1/feedback_histories_controller.rb

```diff
@@ -94,4 +94,12 @@ def destroy
       ]
     )[:feedback_histories]
   end
+
+  private def rename_activity_session_uid_params_key(params_hash)
+    # We've changed the name of the model param in FeedbackHistory, but don't (currently)
```

Comment:
Since this is a product that hasn't been released to the public yet, it's probably better to bite the bullet and update the Feedback APIs to use the new field name vs. maintaining backwards compatibility.

So I'll leave it up to you, you can roll out this code as is and do the name migration is a separate PR, or update this PR.

Reply:
I did a quick audit, and it turns out that nothing reads this at the moment.